PubSub/tests/System/PublishAndPullTest.php

```diff
@@ -77,10 +77,12 @@ public function testPublishMessagesAndPull($client)
         $sub->modifyAckDeadlineBatch($actualMessages, 15);
         $sub->acknowledgeBatch($actualMessages);
 
-        $this->assertEquals($messages[0]['data'], $actualMessages[0]->data());
-        $this->assertEquals($messages[0]['attributes'], $actualMessages[0]->attributes());
-        $this->assertEquals($messages[1]['data'], $actualMessages[1]->data());
-        $this->assertEquals($messages[1]['attributes'], $actualMessages[1]->attributes());
+        $data = [$actualMessages[0]->data(), $actualMessages[1]->data()];
+        $attributes = [$actualMessages[0]->attributes(), $actualMessages[1]->attributes()];
+        $this->assertTrue(in_array($messages[0]['data'], $data));
```

Comment:
Was there a reason to remove `assertEquals` in favor of an `in_array` assert?
It seems fine before as well.

Reply:
Actually, I was getting the pulled messages in an unordered fashion. When I run this alone, everything works, but when I run the full system tests, it was getting not in order. Then I read this https://cloud.google.com/pubsub/docs/ordering and understand ordering is not really guaranteed, so added changed this way.